sdks/python/apache_beam/examples/wordcount_minimal.py

```diff
@@ -73,22 +73,26 @@ def run(argv=None, save_main_session=True):
   parser.add_argument(
       '--output',
       dest='output',
-      # CHANGE 1/5: The Google Cloud Storage path is required
+      # CHANGE 1/6: The Google Cloud Storage path is required
       # for outputting the results.
       default='gs://YOUR_OUTPUT_BUCKET/AND_OUTPUT_PREFIX',
       help='Output file to write results to.')
   known_args, pipeline_args = parser.parse_known_args(argv)
   pipeline_args.extend([
-      # CHANGE 2/5: (OPTIONAL) Change this to DataflowRunner to
+      # CHANGE 2/6: (OPTIONAL) Change this to DataflowRunner to
       # run your pipeline on the Google Cloud Dataflow Service.
       '--runner=DirectRunner',
-      # CHANGE 3/5: Your project ID is required in order to run your pipeline on
-      # the Google Cloud Dataflow Service.
+      # CHANGE 3/6: (OPTIONAL) Your project ID is required in order to
+      # run your pipeline on the Google Cloud Dataflow Service.
       '--project=SET_YOUR_PROJECT_ID_HERE',
-      # CHANGE 4/5: Your Google Cloud Storage path is required for staging local
+      # CHANGE 4/6: (OPTIONAL) The Google Cloud Engine region is
```

Comment:
See https://cloud.google.com/dataflow/docs/concepts/regional-endpoints

```suggestion
      # CHANGE 4/6: (OPTIONAL) The Google Compute Engine region (such as us-central1) is
```